### PR TITLE
Test final snapshotlogic

### PR DESCRIPTION
**Pull request by rifisdfds**

### CODE REVIEW

File: package/sqldatabases/definition.yaml [hunk 1]
```diff
@@ -93,11 +93,14 @@ spec:
                     pattern: '^([01][0-9]|2[0-3]):([0-5][0-9])-([01][0-9]|2[0-3]):([0-5][0-9])$'
                     default: "00:00-01:00"
                     example: "A valid example: 01:00-02:00"
-                  isNonProduction:
-                    description: Is a non-production database
+                  isProduction:
+                    description: Is a production database
                     type: boolean
+                  finalBackupName:
```
Comment (samidbb) [inline package/sqldatabases/definition.yaml:99]: shouldn't be required?

Comment (rifisdfds) [reply package/sqldatabases/definition.yaml:99]: nope, you don't provide a value for non-production otherwise you get the problem deleting

Comment (samidbb) [reply package/sqldatabases/definition.yaml:99]: oh dear!

Comment (samidbb) [reply package/sqldatabases/definition.yaml:99]: just go ahead with the merge otherwise I will jump off of the root top

**Pull request merged**